src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {
+    booklist: 0,
+    categories: 0,
```

Comment:
- According to the project requirements, you are expected to use the reducers from the `bookSlice` and `categoriesSlice` in the redux store. Please import them and use them in the store 👍

Reply:
Hello @mckent05!

Thank you for the comment! I have solved it now!